problem_builder/templates/html/dashboard.html

```diff
@@ -42,16 +42,30 @@
           {% for mcq in block.mcqs %}
             <tr>
               <th class="desc">{{ mcq.display_name }}</th>
-              <td class="value" {% if mcq.color %}style="border-right-color: {{mcq.color}};"{% endif %}>
-                {% if mcq.value %}{{ mcq.value }}{% endif %}
+              <td class="value"
+                  {% if mcq.color %} style="border-right-color: {{mcq.color}};"{% endif %}
+                  {% if not show_numbers %}
+                    {% if mcq.value %} aria-label="{{mcq.value}}" {% else %} aria-label="{% trans 'No value yet' %}" {%endif%}
```

Comment:
I guess we can wait for Mark's advice on this. My worry is that it's really unclear what this block represents until you stare at it a bit, and if you're not sighted and you can't see the graph, it may be even more unclear, so adding more descriptive aria-labels may help.